test/integration/simple_operations_test.lua

```diff
@@ -394,6 +394,8 @@ pgroup:add('test_upsert', function(g)
 end)
 
 pgroup:add('test_intermediate_nullable_fields_update', function(g)
+    -- Blocked by https://github.com/tarantool/tarantool/issues/6069
+    t.skip_if(_TARANTOOL > "2.8")
```

Comment:
The condition looks strange.

```sh
$ (needle='tuple: fix crash in reversed nullable json update'; for branch in 1.7 1.9 1.10 2.1 2.2 2.3 2.4 2.5 2.6 2.7 2.8 master; do printf "${branch}\t"; commit_description=$(git log --oneline "origin/${branch}" | grep -F "${needle}"); [ -n "${commit_description}" ] && echo "$(git describe --long --always $(echo "${commit_description}" | cut -d" " -f1)) $(echo "${commit_description}" | cut -d" " -f2-)" || echo "(not found)"; done)
1.7	(not found)
1.9	(not found)
1.10	(not found)
2.1	(not found)
2.2	(not found)
2.3	(not found)
2.4	(not found)
2.5	(not found)
2.6	(not found)
2.7	(not found)
2.8	2.8.1-13-g6e0c615e6 tuple: fix crash in reversed nullable json update
master	2.9.0-26-g75b2ccce8 tuple: fix crash in reversed nullable json update
```

It seems it is the regression from the following commit:

```sh
$ (needle='update: allow update absent nullable fields'; for branch in 1.7 1.9 1.10 2.1 2.2 2.3 2.4 2.5 2.6 2.7 2.8 master; do printf "${branch}\t"; commit_description=$(git log --oneline "origin/${branch}" | grep -F "${needle}"); [ -n "${commit_description}" ] && echo "$(git describe --long --always $(echo "${commit_description}" | cut -d" " -f1)) $(echo "${commit_description}" | cut -d" " -f2-)" || echo "(not found)"; done)
1.7	(not found)
1.9	(not found)
1.10	(not found)
2.1	(not found)
2.2	(not found)
2.3	(not found)
2.4	(not found)
2.5	(not found)
2.6	(not found)
2.7	(not found)
2.8	2.8.0-223-g2bb373b9f update: allow update absent nullable fields
master	2.8.0-223-g2bb373b9f update: allow update absent nullable fields
```

So we should skip the test for versions in the following ranges: [2.8.0-223, 2.8.1-13), [2.9.0-0, 2.9.0-26).

Reply:
Yes, currently we could drop this check (it was introduced before 2.8.2 was released)

Reply:
> Yes, currently we could drop this check (it was introduced before 2.8.2 was released)

done